guides/common/modules/proc_renewing-a-custom-ssl-certificate-proxy.adoc

```diff
@@ -0,0 +1,83 @@
+[id="Renewing_a_Custom_SSL_Certificate_{smart-proxy-context}_{context}"]
+= Renewing a {customssltitle} certificate on {SmartProxyServer}
+
+[NOTE]
+====
+If your {SmartProxy} is already configured with CA-signed SSL certificates and has been renewed by the same Certificate Authority, there is no need to replace the consumer package `katello-ca-consumer-latest.noarch.rpm` on clients registered to {SmartProxy}.
+
+If the validity of CA-signed SSL certificates installed on {SmartProxy} from Certificate Authority A expires, and you installed an SSL certificate from Certificate Authority B, then you must re-deploy the `katello-ca-consumer-latest.noarch.rpm` package on every client registered to external {SmartProxyServers}.
+This is because you require a new CA certificate for a client to authenticate or communicate with External {SmartProxies}.
+====
+
+Use this procedure to renew a {customssl} certificate signed by a Certificate Authority for your {SmartProxyServer}.
+The `{foreman-installer}` command, which the `{certs-generate}` command returns, is unique to each {SmartProxyServer}.
+Do not use the same command on more than one {SmartProxyServer}.
+
+.Prerequisites
+* You must create a new Certificate Signing Request and send it to the Certificate Authority to sign the certificate.
+In return, you will receive the {SmartProxyServer} certificate and CA bundle.
+You must keep the {SmartProxyServer} SSL certificates on the {ProjectServer} by creating a folder and verifying them through the `katello-certs-check` command.
+
+.Procedure
+. On {ProjectServer} validate the {customssl} certificate input files:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# katello-certs-check \
+-c _/root/{smart-proxy-context}_cert/New_{smart-proxy-context}_cert.pem_ \
+-k _/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem_ \
+-b _/root/{smart-proxy-context}_cert/ca_cert_bundle.pem_
+----
++
+If the CN has a wildcard value, you must add the `-t {smart-proxy-context}` option to the `katello-certs-check` command.
+
+. Use the following `{certs-generate}` commands to generate the certificate archive file for your {SmartProxyServer}.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+{certs-generate} --foreman-proxy-fqdn "{smartproxy-example-com}" \
+--certs-tar  "_/root/_My_Certificates_/{smartproxy-example-com}-certs.tar_" \
+--server-cert "_/root/_My_Certificates_/New-{smart-proxy-context}_cert.pem_" \
+--server-key "_/root/_My_Certificates_/{smart-proxy-context}_cert_key.pem_" \
+--server-ca-cert "_/root/_My_Certificates_/ca_cert_bundle.pem_" \
+--certs-update-server
+----
+
+. On {ProjectServer}, copy the certificate archive file to your {SmartProxyServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+scp _/root/{smart-proxy-context}_cert/{smartproxy-example-com}-certs.tar_ \
```

Comment:
```suggestion
# scp _/root/{smart-proxy-context}_cert/{smartproxy-example-com}-certs.tar_ \
```

you're also missing the `# ` prefix for commands in other code blocks.

Reply:
Yes!